package.json

```diff
@@ -58,7 +58,7 @@
     "@storybook/html": "^6.2.9",
     "@testing-library/dom": "^7.31.2",
     "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/react": "^11.2.7",
+    "@testing-library/react": "^12.0.0",
```

Comment:
Surely we don't need this? We don't have any react components. Is this a storybook thing?